### DISCUSSION TITLE
How might I add the ability to close the app via cmd+q on macos?

### DISCUSSION THREAD

**Discussion by dominikwilkowski** [Relm4/Relm4#572] [Q&A]
I noticed that all examples (At least I tried about 90%) don't close on <kbd>cmd</kbd> + <kbd>q</kbd> as I would expect on macOS? Any pointers here to the docs bit I missed?

**Comment by AaronErhardt** [comment]
If you want to handle shortcuts, you have to add them manually (neither GTK nor Relm4 add such a shortcut). You can do this by setting up an action and registering a handler that closes the app, similar to [this example](https://github.com/Relm4/Relm4/blob/main/examples/menu.rs#L127) but with `"<Control>Q"` instead.

**Reply by dominikwilkowski** [reply]
I could be wrong but this may be related to the global menu as well. Instead of adding an action you should just enable the quit menu item that by default comes with the shortcut? It's currently disabled which seems to be the bug outlined here: https://github.com/orgs/Relm4/discussions/550

<img width="212" alt="image" src="https://github.com/Relm4/Relm4/assets/1266923/403e5c14-9f04-4ac5-a76a-7a4448c070a4">

**Reply by AaronErhardt** [reply]
> Instead of adding an action you should just enable the quit menu item that by default comes with the shortcut

Enabling a menu item can only be done by actions and actions don't work as a shortcut unless you set an accelerator key (that's how GTK calls it) for them.
